build/legacy-mongo-shell/test.example.js

```diff
@@ -0,0 +1,29 @@
+// Move to test.js and run with `task mongo-test`.
+
+(function() {
+    "use strict";
+
+    const col = db.test;
+
+    col.drop();
+
+    const init = [
+        { _id: "double", v: 42.13 },
+        { _id: "double-whole", v: 42.0 },
+        { _id: "double-zero", v: 0.0 },
+    ];
+
+    col.insertMany(init);
+
+    const query = { v: { $gt: 42.0 } };
+
+    const expected = [
+        { _id: "double", v: 42.13 },
+    ];
+
+    const actual = col.find(query).toArray();
+
+    assert.eq(expected, actual);
+
+    print("test.js passed!")
+})();
```

Comment:
We could use the [eslint](https://eslint.org/) for a linter if required, i.e. we expect to add more files, otherwise ignore. I use the google style guide.

Just run `npx eslint test.js --fix` once configured to fix a file.

See https://github.com/b1ron/FerretDB/blob/test-script/build/legacy-mongo-shell/test.example.js.

Reply:
#1667